### PR TITLE
feat: add io home inline callout to consul.io

### PR DESCRIPTION
**Pull request by alexcarpenter**
## ✅ "Ready for Review" checklist

- [x] The Vercel preview link has been added to this PR's description
- [x] The Asana task has been added to this PR's description
- [ ] This PR's link has been added to the "📐 Pull Request" field on the Asana task
- [ ] This Vercel preview link has been added to the "🔍 Preview" field on the Asana task
- [ ] The description template has been filled in below

---

## 🔗 Relevant links

- [Preview link](https://dev-portal-git-acio-inline-callout-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1200641486648086/1202192219912329) 🎟️
- [Dato record](https://io-marketing-pages.admin.datocms.com/editor/item_types/1510835/items/82222195/edit)

## 🤷 Why

Adds inline callout option to consul homepage.

## 📸 Design Screenshots

CleanShot 2022-05-19 at 14 00 31@2x

### CODE REVIEW

File: src/components/_proxied-dot-io/common/io-home-inline-callout/style.module.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.inlineCallout {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 32px;
+  gap: 16px 32px;
+  background-color: var(--brand-secondary);
```
Comment (EnMod) [inline src/components/_proxied-dot-io/common/io-home-inline-callout/style.module.css:8]: ⛏️ Hmm is there a lighter variant of `--brand-secondary` that could be used? Just noticing the background in preview is darker than screenshots in the Asana thread:

chrome_dZVx2NReks

Or would changing this adversely affect other product pages?

Comment (alexcarpenter) [reply src/components/_proxied-dot-io/common/io-home-inline-callout/style.module.css:8]: Yeah, goal is to use `--brand-secondary` to match the other elements on the page

CleanShot 2022-05-19 at 15 41 36@2x

CleanShot 2022-05-19 at 15 42 02@2x

Comment (EnMod) [reply src/components/_proxied-dot-io/common/io-home-inline-callout/style.module.css:8]: Ah yes, good call!

**Review by EnMod — APPROVED** [review]
LGTM!

**Pull request merged**